layouts/docs/single.html

```diff
@@ -16,7 +16,7 @@
                 {{ partial "docs/toc.html" . }}
             </div>
 
-            <div class="md:w-7/12">
+            <div class="md:w-8/12">
```

Comment:
Why'd this change? I think we want this to remain 7.

Reply:
Yes, sorry. Was playing around and thought I'd checked out that file but somehow it got committed.